### DISCUSSION TITLE
Is it possible to change the markerscale of only one of the legends?

### DISCUSSION THREAD

**Discussion by liuzj039** [mwaskom/seaborn#3168] [Q&A]
I want the size of the points in the color's legend to be larger. I know this can be achieved with the `theme`, but it seems to mean that the other legends are also changed. I was wondering if it is possible to control the size of the color legend only.

```
tips = sns.load_dataset("tips")
(
    so.Plot(tips, x='day', y='time', color='total_bill', pointsize='size')
    .add(so.Dots())
    .scale(pointsize=so.Continuous(values=(0, 15), norm=(0,50)),)
)
```
![image](https://user-images.githubusercontent.com/55271666/204175113-f56efa5e-b352-4d91-8582-de7dc0caee3f.png)
```
(
    so.Plot(tips, x='day', y='time', color='total_bill', pointsize='size')
    .add(so.Dots())
    .scale(pointsize=so.Continuous(values=(0, 15), norm=(0,50)),)
    .theme({'legend.markerscale':1.5})
)
```
![image](https://user-images.githubusercontent.com/55271666/204175158-a6646c84-f49b-4176-a8f3-abed644f9197.png)

**Comment by mwaskom** [comment]
Good question. Not currently possible, I don't think, but you could use the `Dot` mark to take advantage of its larger default pointsize

```
tips = sns.load_dataset("tips")
(
    so.Plot(tips, x='day', y='time', color='total_bill', pointsize='size')
    .add(so.Dot())
    .scale(pointsize=so.Continuous(values=(0, 15), norm=(0,50)),)
)
```
![image](https://user-images.githubusercontent.com/315810/204404406-3a3a51fd-dfb8-4168-a8e6-e64231685d03.png)

Set `alpha=.2, edgealpha=1` to more closely match `Dots`.

**Comment by liuzj039** [comment]
Thank you for the detailed explanation !
